### PR TITLE
Add tag management

### PR DESCRIPTION
**Pull request by KushalP**
This is PR is a large change and probably requires reviewing by @alext and @rboulton. It moves the tag management UI into Collections Publisher, keeping Panopticon up-to-date with any changes as they happen.

Worth saying that this is largely work done by @JordanHatch (thanks!), but the final two commits from me are to close off the story.

Story: https://www.pivotaltracker.com/n/projects/1110932/stories/79178100

### CODE REVIEW

File: config/initializers/services.rb [hunk 1]
```diff
@@ -22,3 +22,6 @@ class ServiceNotRegisteredException < Exception; end
 
 require 'gds_api/publishing_api'
 CollectionsPublisher.services(:publishing_api, GdsApi::PublishingApi.new(Plek.new.find('publishing-api')))
+
+require 'gds_api/panopticon'
+CollectionsPublisher.services(:panopticon, GdsApi::Panopticon.new(Plek.new.find('panopticon'), bearer_token: 'example'))
```
Comment (alext) [inline config/initializers/services.rb]: We'll need some mechanism to specify the real bearer token at deploy time, ideally without overwriting this whole file.

As an aside, the `services` method definition above really doesn't belong in here, but that's unrelated to this PR.

**Comment by alext** [comment]
:+1: Looks good to me.  I'll let @rboulton give this a once-over as well...

**Comment by elliotcm** [comment]
Happy to see this completed :star2:

**Comment by rboulton** [comment]
:fireworks: :100:

**Comment by JordanHatch** [comment]
Fantastic, thanks @KushalP! :palm_tree:

**Pull request merged**